src/components/chart/chart.directive.coffee

```diff
@@ -1,73 +1,40 @@
 # chart.js charting attribute directive.
 angular
   .module('impac.components.chart',[])
-  .directive('impacChart', ($timeout, $log) ->
+  .directive('impacChart', ($log) ->
     return {
       restrict: 'A',
       scope: {
         drawTrigger: '='
         deferred: '='
       },
-      template: '<canvas height="1" width="1"></canvas>',
       link: (scope,elem,attr) ->
-        options = {
-          bezierCurve: true,
-          pointDotRadius : 3,
-          responsive: true,
-          scaleShowLabels : true,
-          scaleShowLabelBackdrop : true,
-          scaleBeginAtZero: true,
-          scaleShowGridLines: true,
-        }
-
         # Draw a chart in the canvas
         # (ChartJs way of drawing a chart is to create a new Chart() element in the canvas context)
         # ------------------------------------
-        scope.draw = (data) ->
-          if !_.isEmpty(data.options)
-            angular.extend(options,data.options)
+        scope.draw = (chartData) ->
+
+          angular.merge(chartData.options, {
+            scales: {
+              xAxes: [{
+                display: false
+              }]
+            }
+          }) if chartData.options.showXLabels? && !chartData.options.showXLabels
```

Comment:
Can you not "inline" if on multiple lines instruction? On a block like this the if may be missed.